src/safeds/ml/nn/_model.py

```diff
@@ -0,0 +1,186 @@
+import numpy as np
+import torch
+from torch import nn
+
+from safeds.data.tabular.containers import TaggedTable
+from safeds.exceptions import ClosedBound, OutOfBoundsError
+from safeds.ml.nn._fnn_layer import FNNLayer
+
+
+class RegressionModel:
```

Comment:
Use the [Regressor](https://library.safeds.com/en/stable/reference/safeds/ml/classical/regression/Regressor/) abstract class as the baseline for your interface.